### DISCUSSION TITLE
Larger text display

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5942] [Ideas and suggestions]
A larger text display that could display an entire paragraph at once could enable some interesting new ways of rendering information to submariners

**Comment by Veritius** [comment]
Alternatively: Scalable text display with newline support.

**Comment by MaaiG** [comment]
A variable-size text display is needed. Small - to display numbers. Large for other information. Also, it would be good if there was a backlighting for the "sign".

**Comment by MaaiG** [comment]
I’m still hoping for a shorter-length text display to indicate numbers.
Now, there is no diversity at all. This seriously limits the design solutions.
![Image 001_21-03](https://user-images.githubusercontent.com/93348499/196509912-d0576406-eb6e-4693-ae39-98f34e7eed54.png)
